src/Filters/molecules/FilterDropdown.tsx

```diff
@@ -339,7 +334,7 @@ const FilterDropdown: React.FC<IFilterDropdown> = ({
             : (
               <ResultsContainer>
                 {hasOptionsFilter && <ResultCounter>{getResultText(searchResultText, visibleList.length, list.length)}</ResultCounter>}
-                <OptionList>
+                <OptionList hideVerticleScrollbar={list.length < 5 || list.length === 5}>
```

Comment:
I think its not needed, we have max-height in OptionList and that should be the only criteria for showing scroll-bar, let css decide whether to display scroll or not based upon max-height, you can use `overflow: auto;`.

Reply:
Please also style the scroll-bar to match ui-kit's styling.
Here is one example which you can refer.
![image](https://github.com/future-standard/scorer-ui-kit/assets/95204740/4ac4199c-fe59-484f-bb4a-073fa22d9c91)